package.json

```diff
@@ -5,16 +5,20 @@
   "author": "Brandon Keown <bkeown@blitz.io>",
   "description": "A simple helpful robot for your Company",
   "scripts": {
+    "coverage": "istanbul cover --include-all-sources --root lib _mocha -- --reporter nyan 'spec/**/*.spec.js'",
     "hubot": "hubot --adapter slack",
-    "test": "mocha --watch spec/**/*.spec.js --reporter min",
-    "coverage": "istanbul cover --include-all-sources --root lib _mocha -- --reporter nyan 'spec/**/*.spec.js'"
+    "lint": "eslint .",
+    "pre-commit-msg": "echo 'Pre-commit linting and unit testing' && exit 0",
+    "test": "mocha spec/**/*.spec.js --reporter min",
+    "test-watch": "mocha --watch spec/**/*.spec.js --reporter min"
```

Comment:
I don't know what the dot reporter is. You set all that up :D, I don't really use it. I am amenable to changing it for the pre-commit stuff.